include/spirv-tools/optimizer.hpp

```diff
@@ -230,6 +230,23 @@ Optimizer::PassToken CreateInlinePass();
 // by LocalSingleStoreElim and LocalSSARewrite.
 Optimizer::PassToken CreateLocalSingleBlockLoadStoreElimPass();
 
+// Creates an SSA local variable load/store elimination pass.
+// For every entry point function, eliminate all loads and stores of function
+// scope variables only referenced with non-access-chain loads and stores.
+// Eliminate the variables as well. 
+//
+// The presence of access chain references and function calls can inhibit
+// the above optimization.
+//
+// Only shader modules with logical addressing are currently processed.
+// Currently modules with any extensions enabled are not processed. This
+// is left for future work.
+//
+// This pass is most effective if preceeded by Inlining and 
+// LocalAccessChainConvert. LocalSingleStoreElim and LocalSingleBlockElim
+// will reduce the work that this pass has to do.
+Optimizer::PassToken CreateLocalSSAElimPass();
```

Comment:
My thinking here was to utilize a similar naming mechanism for the other previous similar phases: --eliminate-local-<method>, where method could be single-store or single-block. How about --eliminate-local-multiple-store?